torch/_inductor/codegen/triton.py

```diff
@@ -2362,6 +2363,7 @@ def define_kernel(self, src_code, node_schedule):
             # use the original src_code as the key
             wrapper.src_to_kernel[src_code] = kernel_name
             subs_name = kernel_name if config.triton.unique_kernel_names else "triton_"
+            src_code = src_code.replace("DESCRIPTIVE_KRNL_NAME", kernel_name)
```

Comment:
Maybe worth adding a note that `DESCRIPTIVE_KRNL_NAME` is just used in torch profiler, so we can use full `kernel_name` without breaking compilation caching... at first i was a bit confused that we had gone to pains to avoid using `kernel_name` on the line above but then just used it anyway on this line.